config.json

```diff
@@ -490,6 +490,14 @@
         "prerequisites": [],
         "difficulty": 7
       },
+      {
+        "slug": "binary-search",
+        "name": "Binary Search",
+        "uuid": "0a702019-2325-4124-862a-49cc5615cee4",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 7
```

Comment:
This was harder than I anticipated

Reply:
Your solution looks straightforward though. I guess the pattern matching was the difficult part?

Reply:
I found the stack juggling the hardest part